cli/src/commands/mod.rs

```diff
@@ -3020,8 +3021,16 @@ fn description_template_for_cmd_split(
         &EverythingMatcher,
         &[DiffFormat::Summary],
     )?;
-    Ok(format!("JJ: {intro}\n{overall_commit_description}\n")
-        + &diff_summary_to_description(&diff_summary_bytes))
+    let description = if overall_commit_description.is_empty() {
+        command
+            .settings()
+            .config()
+            .get_string("ui.default-description")
+            .unwrap_or("".to_owned())
```

Comment:
I think you can use `unwrap_or_default()` here. You can also move these last few lines into a new `UserSettings::default_description()` and reuse that in the other place